package/MDAnalysis/core/groups.py

```diff
@@ -2977,7 +2977,15 @@ def select_atoms(self, sel, *othersel, periodic=True, rtol=1e-05,
             smarts *SMARTS-query*
                 select atoms using Daylight's SMARTS queries, e.g. ``smarts
                 [#7;R]`` to find nitrogen atoms in rings. Requires RDKit.
-                All matches (max 1000) are combined as a unique match
+                All matches are combined as a unique match. Uses two sets of
+                kwargs: rdkit_kwargs is passed to `RDKitConverter.convert()`
+                and smarts_kwargs is passed to RDKit's [GetSubstructMatches](
```

Comment:
I don't think the formatting is working here (https://mdanalysis--3470.org.readthedocs.build/en/3470/documentation_pages/core/groups.html?highlight=select_atoms#MDAnalysis.core.groups.AtomGroup.select_atoms). Please check and fix. Also might be worth re-organising / re-writing this to be a bit more readable.

Reply:
working on it...